plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -650,6 +655,36 @@ public TableStatistics getTableStatistics(ConnectorSession session, JdbcTableHan
         }
     }
 
+    @Override
+    public JdbcProcedureHandle getProcedureHandle(ConnectorSession session, ProcedureInformation procedureInformation)
+
+    {
+        if (procedureInformation.inputArguments().stream().anyMatch(argument -> argument.contains(";"))) {
+            throw new TrinoException(INVALID_ARGUMENTS, "InputArgument doesn't support special characters like ';'");
```

Comment:
Should we also escape all single quotes (and `[`/`]`)?

Reply:
I don't think it is required from the client side - user could escape them as a part of their input arguments. WDYT ?

Reply:
That's a good point. Table functions use syntax native to the system so we don't perform any extra processing on them intentionally.